src/index.js

```diff
@@ -4,9 +4,18 @@ import ReactDOM from "react-dom"
 import App from "containers/App/App"
 import SynthApp from "./synth"
 
+function renderApp() {
+  ReactDOM.render(<App />, document.querySelector("#root"))
```

Comment:
__[JSHint]__

Expected an assignment or function call and instead saw an expression.


<!-- Comment by [SideCI](https://sideci.com) -->